src/config/defaultConfig.js

```diff
@@ -23,7 +23,7 @@ const getAccountInfo = (account) => accountInfo[account] || accountInfo.default;
 const config = {
   supportEmail: getAccountInfo(accountId).supportEmail,
   pipelineVersionToRerunQC: 2,
-  workerVersion: 4, // needs to match workerVersion in API
+  workerVersion: 5, // needs to match workerVersion in API
```

Comment:
**can be removed before merging** Changed worker version for the staging tests, can be removed before merging